src/styles/_catalog.scss

```diff
@@ -1,9 +1,12 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
I would prefer to create a separate file with variables and set the size of the cards through variables

Reply:
\+